app/src/main/java/com/example/photofy/RecommendationsService.java

```diff
@@ -20,15 +20,14 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 public class RecommendationsService {
 
     public static final String TAG = "RecommendationsService";
 
     private StringBuilder endpoint = new StringBuilder("https://api.spotify.com/v1/recommendations?market=US");
-    private List<Song> songs = new ArrayList<>();
+    private ArrayList<Song> songs = new ArrayList<>();
```

Comment:
The recommendations callback takes in an ArrayList of songs since when passing the songs via an intent later on, I put the songs in putParcelableArrayListExtra, and there is no option for a Parcelable List